pilot/pkg/bootstrap/options.go

```diff
@@ -153,6 +153,7 @@ func (p *PilotArgs) applyDefaults() {
 	p.KeepaliveOptions = keepalive.DefaultOption()
 	p.RegistryOptions.DistributionTrackingEnabled = features.EnableDistributionTracking
 	p.RegistryOptions.DistributionCacheRetention = features.DistributionHistoryRetention
+	p.ShutdownDuration = 10 * time.Second
```

Comment:
nit: it seems we never have a flag setting ShutdownDuration

Reply:
yes, the ShutdownDuration parameter is then added to the flag, and some scenarios do need to configure this parameter.

Reply:
I have already added a flag setting ShutdownDuration in pilot